### DISCUSSION TITLE
Looking for help and hints to train Tortoise model

### DISCUSSION THREAD

**Discussion by TheBlueSky** [neonbjb/tortoise-tts#569] [Q&A]
First of all, I would like to thank [James Betker](https://github.com/neonbjb), who started this project, and to everyone who contributed and contributes to this project, especially [Manmay Nakhashi](https://github.com/manmay-nakhashi).

I would like to train a model to be able to use it for non-Latin language TTS. Many of the online tutorials are about fine tuning the built in model for specific voices, and I was unable to find what helps me to train a model.

The closest I got is [this](https://github.com/neonbjb/tortoise-tts/discussions/129) discussion, but it only talked on a very high-level that many hours of audio and its transcript is needed, which are the 2 things that I already know are needed :)

What I am looking for is something like the tools, the datasets, whether I need language-specific tokenizer, and so on.

I appreciate any help and hint that point me to the right direction.

**Comment by jjjota23** [comment]
How can we train in Portuguese? I have a source with more than 50,000 hours of audio, clean and of very high sound quality.

**Comment by CoderCowMoo** [comment]
Hi I'm still learning and curious about this. Since finetuning tortoise is the same as training for the implementations available, wouldn't it be the same to finetune tortoise on this data? They already use whisper for transcription and dataset generation. I am referring to the [mrq](https://git.ecker.tech/mrq/ai-voice-cloning) repo btw.

**Reply by TheBlueSky** [reply]
I am not sure I understood your comment. Finetuning is a much simpler process than creating a model/checkpoint from scratch.

For example, if I want to train an Arabic model, I want to know what tools I need (of course, in addition to the dataset) to be able to start the training process. Those tools can be the transcription tools, if the transcript is not available, the tokenizers for the language, the training software, and so on, in addition to the steps to follow.

I am not necessarily looking for a step by step guide, although that would be awesome, but any resources that I can read and learn from the method to train such models.
